src/frontends/pytorch/src/utils_quantize.cpp

```diff
@@ -0,0 +1,264 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "utils_quantize.hpp"
+
+#include "openvino/frontend/pytorch/node_context.hpp"
+#include "openvino/op/broadcast.hpp"
+#include "openvino/op/convert.hpp"
+#include "openvino/op/fake_quantize.hpp"
+#include "openvino/op/maximum.hpp"
+#include "openvino/op/minimum.hpp"
+#include "openvino/op/multiply.hpp"
+#include "openvino/op/reshape.hpp"
+#include "openvino/op/round.hpp"
+#include "openvino/op/scatter_elements_update.hpp"
+#include "openvino/op/subtract.hpp"
+
+#include "openvino/op/convert_like.hpp"
+#include "openvino/op/divide.hpp"
+#include "openvino/op/add.hpp"
+#include "openvino/op/clamp.hpp"
+
+
+namespace ov {
+namespace frontend {
+namespace pytorch {
+
+using namespace ov::op;
+
+ov::Output<ov::Node> quantize(const NodeContext& context,
+                                   std::shared_ptr<ov::Node> input,
+                                   std::shared_ptr<ov::Node> scale,
+                                   std::shared_ptr<ov::Node> zero_point,
+                                   std::shared_ptr<ov::Node> axis,
+                                   ov::element::Type dtype,
+                                   QuantizedPtNodeType quantization_type) {
```

Comment:
```suggestion
ov::Output<ov::Node> quantize(const NodeContext& context,
                                   const Output<Node>& input,
                                   const Output<Node>& scale,
                                   const Output<Node>& zero_point,
                                   const Output<Node>& axis,
                                   ov::element::Type dtype,
                                   QuantizedPtNodeType quantization_type) {
```
Please do not use node where you can use outputs

Reply:
Changed to Output!